Made consistent with index.html.

rogerharkavy opened the pull request:
Just reformatted intro text. Addresses #27

rogerharkavy committed:
Made consistent with index.html.

Just reformatted intro text.

yetzt commented:
why would you replace a semantic heading with a deprecated-bold text in a paragraph?

rogerharkavy commented:
Force of habit. Feel free to change it if you think it should be done differently.

rogerharkavy committed:
Changed from <p> to <h2>.

Changed header line from <p> to <h2> as was suggested in pull request discussion.

rogerharkavy commented:
Changed the HTML as per the suggestion.
